polyfills/IntersectionObserver/polyfill.js

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
If this is published on NPM then we could use our update tasks to pull in the polyfill